src/System Application/App/Json/src/Json.Codeunit.al

```diff
@@ -0,0 +1,64 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Json;
+
+codeunit 5460 Json
```

Comment:
I'd like the developer tools team to take a look to see if these can be added to the language. We shouldn't need wrapper functions to use built in json objects.

Reply:
I'll ping them and ask for a timeline for this. If this cannot be done "swiftly", we would this as a work around.

Reply:
Long story short: Developer Tools won't have time to deliver this on short notice. They said that they've previously had to down prioritize work on JSON support due to lack of requests from internal team and partners.

Making a wrapper isn't wrong per se and could be replaced by language support, when we get around to it.

We'll hence continue with this module! I hope I'll find some time to look at it tomorrow.

Reply:
Thanks.
Added new Commit with additional functions https://github.com/microsoft/BCApps/pull/716/commits/88ea0dd3821dc3775a2dc9a2cfd90a6941aab0f4

Tests added as well
<img width="1296" alt="image" src="https://github.com/microsoft/BCApps/assets/29101017/1745d11b-2375-4dd7-8d8c-a483a7becc92">